src/fsharp/IlxGen.fs

```diff
@@ -6418,7 +6458,9 @@ and GenTypeDef cenv mgbuf lazyInitInfo eenv m (tycon:Tycon) =
                  // Records that are value types do not create a default constructor with CLIMutable or ComVisible
                  if not isStructRecord && (isCLIMutable || (TryFindFSharpBoolAttribute cenv.g cenv.g.attrib_ComVisibleAttribute tycon.Attribs = Some true)) then
                      yield mkILSimpleStorageCtor(None, Some cenv.g.ilg.typ_Object.TypeSpec, ilThisTy, [], reprAccess) 
-
+                 
+                 if (not <| tycon.HasMember cenv.g "ToString" []) then
```

Comment:
We generally don't use the backpipe operator in the compiler (except ``xyz <| (fun a -> ...)`` now and then)